.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,24 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
```

Comment:
we should eventually just bind to LTS

Reply:
Hmm, maybe this will follow the nvm input, I'll try it

Reply:
```suggestion
        node-version: [8.x, lts/*, node]
```

Reply:
Any reason to keep testing on 8?

Reply:
I think "lts" and "current" is fine. We shouldn't hold ourselves back unless we really need to.

Reply:
It's just there because no need to remove it yet, as we haven't seen a failure there. I think we can freely remove it once we see one.